pkg/package/service/config.go

```diff
@@ -39,29 +38,73 @@ func (ps packageService) ImportPackageFromConfig(path string) (*domain.Package,
 	return pkg, nil
 }
 
-func (ps packageService) ExportPackageToConfig(pkg domain.Package, destination string) (string, error) {
-	confName := filepath.Join(destination, "proji-"+pkg.Name+".toml")
+func (ps packageService) ExportPackageToConfig(pkg domain.Package, destination string, asJson bool) (string, error) {
+	fileExtension := ".toml"
+	if asJson {
+		fileExtension = ".json"
+	}
+	confName := filepath.Join(destination, "proji-"+pkg.Name+fileExtension)
 	conf, err := os.Create(confName)
 	if err != nil {
 		return confName, err
 	}
 	defer conf.Close()
-	return confName, toml.NewEncoder(conf).Order(toml.OrderPreserve).Encode(pkg)
+
+	if !asJson {
+		return confName, ps.getTomlEncoderFunction(conf)(pkg)
+	}
+	return confName, ps.getJsonEncoderFunction(conf)(pkg)
+}
+
+func (ps packageService) getTomlEncoderFunction(conf *os.File) func(v interface{}) error {
+	return toml.NewEncoder(conf).Order(toml.OrderPreserve).Encode
+}
+
+func (ps packageService) getJsonEncoderFunction(conf *os.File) func(v interface{}) error {
+	return json.NewEncoder(conf).Encode
```

Comment:
One last thing mate. Just noticed while testing your changes that `json` exports are not formatted. The file is just one line of json with everything in it. The config should be human-readable too. So, we need to add indentation to the json encoder like this: 

	encoder := json.NewEncoder(conf)
	encoder.SetIndent("", "  ")
	return encoder.Encode

Reply:
sure thing